src/components/ModelGroupToggle/ModelGroupToggle.js

```diff
@@ -1,11 +1,18 @@
-import React from "react";
+import React, { useEffect } from "react";
+import { useHistory } from "react-router-dom";
 import classNames from "classnames";
 
 import "./_model-group-toggle.scss";
 
 const buttons = ["status", "cloud", "owner"];
 
 const ModelGroupToggle = ({ groupedBy, setGroupedBy }) => {
+  const history = useHistory();
+
+  useEffect(() => {
+    history.push(`/models?groupby=${groupedBy}`);
```

Comment:
This isn't scalable and won't work with other components interacting with the url. You'll want to instead have it just update the `groupby` query parameter. 

We almost always have a `groupedby` value but we don't need to define a query param if it's grouped by the default value.